pkg/update/update_cockroach_version_common.go

```diff
@@ -32,9 +34,17 @@ import (
 // a function which takes a statefulset and returns the same statefulset, with
 // the CockroachDB container image within changed to the new cockroachImage.
 func makeUpdateCockroachVersionFunction(
-	cockroachImage string,
+	cockroachImage, version, oldVersion string,
 ) func(sts *v1.StatefulSet) (*v1.StatefulSet, error) {
 	return func(sts *v1.StatefulSet) (*v1.StatefulSet, error) {
+		timeNow := metav1.Now()
```

Comment:
Is there a reson that this isn't `time.Now()`

Reply:
not really, usually if we expose on the api we use this struct... it s more related to the fact that I  will probably need to add it in the status field, but probably in a future task